### DISCUSSION TITLE
Camera depth of field, focal point, and other queries.

### DISCUSSION THREAD

**Discussion by Inkletblot** [openhsi/openhsi#44] [Q&A] [answered]
Firstly, thank you for this amazing project, both the software and the hardware are awesome.

I'm using the Lucid Camera model (OpenHSI-11) for work on my Honors Project and I have a couple of simple queries I hope you can assist me with.

Firstly, I cannot seem to find mention of a depth of field and focal point for the camera. Essentially, my situation is that I'm using the camera to capture images of objects at a range of about 2m. I am worried that this is too close to the camera. Is 2m too close a target? If yes, what distance should I be looking at capturing from?

The other question I have is potentially a stupid one, however I find myself confused and must ask. Is ELC required to calculate surface reflectance, or can I just use the radiative transfer model Model6SV with the correct settings at time of capture? I seem to have confused myself reading up on calculating surface reflectance and reading the openhsi documentation (which has been a most invaluable resource to me so far).

Many thanks for your time.

Solomon.

P.S. If I record my images as simple digital numbers, can I post-process them in the way mentioned above if I have recorded all of the correct information for the 6S process?

**Comment by christhechris** [comment, marked as answer]
Hi @Inkletblot,

1. The RobotX sensors we delivered focused for around 50-60m or so. It can be refocused, easy way to do this is pointing at something with a alternating dark light pattern (brick wall or piece of paper with black lines etc). I would liaise with the RobotX team before trying to do this though.
2. ELC vs 6SV - You do not need to do both, but ELC is good way to verify your model etc. Both are different ways of estimating how much light hit the ground (or thing observed), with ELC you have something with a know reflectance in your field and can thus work back from that. With the 6SV model you get information abut the environment and simulate the atmospheric column etc.

While in development and figuring things out I definitely recommend recording raw data (level -1 or 0) and post processing (-1 is easiest I think).

**Reply by Inkletblot** [reply]
Fantastic, many thanks.
